lib/geokit-rails/acts_as_mappable.rb

```diff
@@ -93,7 +93,10 @@ module ClassMethods
       # A proxy to an instance of a finder adapter, inferred from the connection's adapter.
       def adapter
         @adapter ||= begin
-          require File.join('geokit-rails', 'adapters', connection.adapter_name.downcase)
+          unless Adapters.const_defined?(connection.adapter_name.camelcase)
+            filename = connection.adapter_name.downcase
+            require File.join('geokit-rails', 'adapters', filename)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.